### PR TITLE
post about CDN

### PR DESCRIPTION
**Pull request by xsuchy**

### CODE REVIEW

File: _posts/2020-02-04-copr-cdn.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    copr-be.cloud.fedoraproject.org
+
+is now accessible using
+
+    download.copr.fedorainfracloud.org
+
+The original URL is and will be available. You can still use it. The second one is CNAME for the CDN and should be much
+faster for you.
+
+The CDN is automatically enabled for new projects. Thou, if you already have enabled some copr repository on your
```
Comment (FrostyX) [inline _posts/2020-02-04-copr-cdn.md]: s/Thou/Though

File: Gemfile [hunk 2]
```diff
@@ -9,8 +9,8 @@ ruby RUBY_VERSION
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
-gem "jekyll", "3.3.0"
-
+gem "jekyll", ">= 3.6.3"
+i
```
Comment (FrostyX) [inline Gemfile]: Possibly a typo by switching to insert mode while already being in insert mode? :D

**Comment by FrostyX** [comment]
Two typos, otherwise LGTM

File: _posts/2020-02-04-copr-cdn.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+workstation/server, it will still use the old URL.
+It is fine; it will continue to work.
+If you want to enable CDN for your repos you have to run:
+
+    dnf copr remove some/project
```
Comment (praiskup) [inline _posts/2020-02-04-copr-cdn.md]: The remove step is redundant here, you can just re-enable.

**Comment by xsuchy** [comment]
Fixed.

**Comment by praiskup** [comment]
Please merge, I can not.

**Pull request merged**